### ISSUE TITLE
Action fails when triggered by a tag.

### ISSUE THREAD

**Issue by rosiel** [SvanBoxel/gitlab-mirror-and-ci-action#24]
We are using the `FOLLOW_TAGS` option and have set up our Github workflow to trigger on tag creation.

The result is, after creating tag `2.9.0`, 

```
error: pathspec '.9.0' did not match any file(s) known to git
fatal: ref HEAD is not a symbolic ref
```

I suspect that this is caused by line 25, `git checkout "${GITHUB_REF:11}"`. When the `GITHUB_REF` is `refs/heads/<branch_name>` this works, but when it's `refs/tags/<tag_name>` it cuts the first letter off the tag name.

I think this could be solved by using `$GITHUB_REF_NAME` which is "The short ref name of the branch or tag that triggered the workflow run. This value matches the branch or tag name shown on GitHub. For example, `feature-branch-1`" Source:  https://docs.github.com/en/actions/learn-github-actions/variables

**Comment by lwih** [comment]
I upvote this issue as I'm facing the same one.
@rosiel - did you find a solution?

**Comment by rosiel** [comment]
No, I'm hoping @SvanBoxel has a chance to look at this - the solution should be as simple as switching out `$GITHUB_REF:11` with `$GITHUB_REF_NAME`.

**Comment by SvanBoxel** [comment]
Hi @rosiel. Could you test your solution and open a PR if you're confident it works? Thank you!

**Comment by SvanBoxel** [comment]
Released as pre-release: https://github.com/SvanBoxel/gitlab-mirror-and-ci-action/releases/tag/0.6
